cli/command/cli.go

```diff
@@ -407,9 +418,55 @@ type ClientInfo struct {
 	DefaultVersion  string
 }
 
-// NewDockerCli returns a DockerCli instance with IO output and error streams set by in, out and err.
-func NewDockerCli(in io.ReadCloser, out, err io.Writer, isTrusted bool, containerizedFn func(string) (clitypes.ContainerizedClient, error)) *DockerCli {
-	return &DockerCli{in: NewInStream(in), out: NewOutStream(out), err: err, contentTrust: isTrusted, newContainerizeClient: containerizedFn}
+// NewDockerCli returns a DockerCli instance with all operators applied on it.
+func NewDockerCli(ops ...DockerCliOperator) *DockerCli {
+	cli := &DockerCli{}
+	cli.Apply(ops...)
+	return cli
+}
+
+// WithStandardStreams sets a cli in, out and err streams with the standard streams.
```

Comment:
Perhaps we should move all these options to a separate file (e.g. `cli/command/cli_options.go` or `cli/command/options.go`)

Reply:
I'll do that 👍